index.js

```diff
@@ -17,13 +26,65 @@ FCM.requestPermissions = () => {
     return FIRMessaging.requestPermissions();
 };
 
+FCM.presentLocalNotification = (details) =>{
+  if (Platform.OS ==='android'){
+      FIRMessaging.presentLocalNotification(details);
+  }
+
+  else if (Platform.OS ==='ios') {
+    const soundName = !details.hasOwnProperty("playSound") || details.playSound === true ? 'default' : '';// empty string results in no sound
+    PushNotificationIOS.presentLocalNotification({
+  			alertBody: details.message,
+  			alertAction: details.alertAction,
+  			category: details.category,
+  			soundName: soundName,
+  			applicationIconBadgeNumber: details.number,
+  			userInfo: details.userInfo
+	   });
+  }
+};
+
+FCM.scheduleLocalNotification = function(details) {
+    var iosNotification = {
+        fireDate: details.date.getTime(),
+        alertBody: details.message,
+        userInfo: details.userInfo,
+        repeatInterval: REPEAT_INTERVAL_IOS[details.repeatEvery] || 0
+    };
+    details.fireDate = details.date.getTime();
+    delete details.date;
+    FIRMessaging.scheduleLocalNotification((Platform.OS === 'ios')? iosNotification : details);
+};
+
+FCM.cancel = (notificationID) => {
+    if(Platform.OS === 'android'){
+        FIRMessaging.cancel(notificationID);
+    }
+};
+
+FCM.cancelAll = () => {
+    if (Platform.OS ==='android'){
+        FIRMessaging.cancelAll();
```

Comment:
can we name this as `cancelLocalNotifications` to be consistent with ios version?